inc/osvr/PluginKit/PluginRegistration.h

```diff
@@ -198,6 +198,15 @@ namespace pluginkit {
         }
     }
     /// @}
+
+    inline void log(OSVR_PluginRegContext ctx, OSVR_LogLevel severity,
+             const char *message) {
+        OSVR_ReturnCode ret = osvrPluginLog(ctx, severity, message);
```

Comment:
I'd rather not have plugins throw exceptions if their logging fails - that seems like a drastic response. ("I couldn't take notes in class... so I pulled the fire alarm!")  Can we just make the logging method never fail?